test/test_plugin_helper.rb

```diff
@@ -4,6 +4,10 @@
 require 'webmock/minitest'
 require 'webmock'
 
+FactoryBot.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
+FactoryBot.definition_file_paths << "#{ForemanFogProxmox::Engine.root}/test/factories" if ForemanBootdisk.with_proxmox?
+FactoryBot.reload
```

Comment:
Out of interest and because I don't know FactoryBot very well: why do you need to reload?

Reply:
To load factories from fog_proxmox otherwise it will use only from foreman and raises error in proxmox compute resource related tests that `Factory not registered: "proxmox_cr"`

Reply:
Makes me wonder if Foreman's test helper should just load all factories from all plugins for you. Might be nice to have in core.

Reply:
I am not so sure about this one as well, but I am not a FactoryBot expert. This could bite tho.